.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-stale: 90
+          days-before-close: 30
+          stale-issue-label: stale
+          stale-issue-message: >
+            This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
+          close-issue-message: >
+            This issue has been closed as it was considered stale, this issue will not be monitored.  If this is a bug and you can reproduce this issue on a [supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and ensure a repro is provided.
+          exempt-issue-labels: discussion,security 🔒,enhancement :sparkles:
+          only-pr-labels: not-a-real-label
```

Comment:
?

Reply:
Only way I can see for it to not close PRs as stale